molior-deploy

```diff
@@ -902,8 +906,9 @@ install_grub_bootloader()
     sed -i "s%^GRUB_CMDLINE_LINUX=.*$%GRUB_CMDLINE_LINUX=\"$LINUX_CMDLINE\"%" $target/etc/default/grub
   fi
   mkdir -p $target/dev/disk/by-uuid
-  touch $target/dev/disk/by-uuid/`blkid  -o value -s UUID $rootpart`
+  cp -a /dev/disk/by-uuid/`blkid  -o value -s UUID $rootpart` $target/dev/disk/by-uuid/
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2006

Reply:
we need a separate PR for this, changing it everywhere. let's be consistent for now...